data-model/fdo-type/term/term.json

```diff
@@ -0,0 +1,184 @@
+      ]
+    },
+    "skos:prefLabel": {
+      "type": "string",
+      "description": "The preferred lexical label for the term",
+      "examples": [
+        "Digital Specimen"
+      ],
+      "@language": {
+        "type": "string"
+      }
+    },
+    "skos:member": {
+  "type": "array",
+  "description": "Indicates that this term is a member of a specific concept scheme or collection",
+  "items": {
+    "type": "string",
+    "format": "uri"
+  },
+  "examples": [
+    ["https://terms.dissco.tech/digital-specimen-terms", "https://vocabularies.dissco.tech/core-terms"]
+  ]
+},
+    "skos:definition": {
+      "type": "string",
+      "description": "A definition of what the term represents.",
+      "examples": [
+        "A digital representation of a physical specimen"
+      ],
+  "@language": {
+        "type": "string"
+      }
+    },
+    "skos:note": {
+      "type": "string",
+      "description": "A note field to capture contexts and other information around the decision.",
+      "examples": [
+        "This term was discussed in projectX"
+      ],
+      "@language": {
+        "type": "string"
+      }
+    },
+  "skos:example": {
+      "type": "string",
+      "description": "An example usage of the term.",
+      "examples": [
+        "ods:physicalSpecimenID is used in Digital Specimen data schema"
+      ]
+    },
+    "dcterms:created": {
+      "type": "string",
+      "description": "The timestamp that the object was created in DiSSCo, following the ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss.SSSXXX",
+      "format": "date-time",
+      "examples": [
+        "2023-10-02T12:31:34.806Z"
+      ]
+    },
+    "dcterms:modified": {
+      "type": "string",
+      "description": "The timestamp that the object was last changed, following the ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss.SSSXXX",
+      "examples": [
+        "2023-10-02T12:31:34.806Z"
+      ]
+    },
+    "dcterms:hasVersion": {
+      "type": "string",
+      "description": "A related resource that is a version, edition, or adaptation of the described resource.",
+      "examples": [
+        "1.0.1"
+      ]
+    },
+    "rdfs:range": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "format": "uri"
+      },
+      "description": "URIs of the classes this term ranges to or belongs to",
+      "examples": [
+        ["https://terms.dissco.tech/digital-specimen-terms#EntityRelationship", "http://purl.org/dc/dcmitype/Collection"]
+      ]
+    },
+    "owl:deprecated": {
+      "type": "boolean",
+      "description": "Indicates whether the term is deprecated",
+      "examples": [
+        false
+      ]
+    }
+  },
+  "required": [
+   "@id",
+   "@type",
+    "ods:version",
+    "ods:fdoType",
+    "dcterms:created",
+    "dcterms:modified", 
+    "dcterms:title",
+    "owl:deprecated",
+    "skos:prefLabel",
+    "skos:definition",
+    "ods:hasAgents",
+    "ods:status",  
+    "rdfs:range"
```

Comment:
Would range always be there? If I got a ods:DigitalSpecimen which is the class than it won't have a range correct?